GetDKAN/dkan
# Import errors and other datastore dashboard improvements

Improvements to the datastore import status page and underlying services:

* Displays datastore import errors on the import dashboard. 
* Improves readability of the overall table
* Fixes bug in pager

## QA Steps

- Enable datastore_mysql_import
- Upload a non-UTF8 CSV file as a resource.
- Run cron
- Visit /admin/dkan/dataset-status
- [ ] Import status should show an error for the non-UTF8 resource
- [ ] Confirm table looks good
- [ ] Confirm pager works as expected

## Known issues

There are still issues with underlying reporting from the fetcher and importer. A file that DKAN refuses to import, such as a non-CSV file, just shows up as "waiting" rather than giving useful feedback. Additionally, imports from the mysql importer ("LOAD INFILE") show as "0%" even though they are done.
